robottelo/constants/__init__.py

```diff
@@ -832,10 +832,12 @@
     'seal-1.0.10-1.noarch.rpm',
 ]
 FAKE_0_YUM_REPO_STRING_BASED_VERSIONS_COUNTS = {'rpm': 35, 'package_group': 2, 'erratum': 4}
-PULP_PUBLISHED_ISO_REPOS_PATH = '/var/lib/pulp/published/http/isos'
-PULP_PUBLISHED_PUPPET_REPOS_PATH = '/var/lib/pulp/published/puppet/https/repos'
 PULP_PUBLISHED_YUM_REPOS_PATH = '/var/lib/pulp/published/yum/http/repos'
```

Comment:
This should be removed too as outdated in 6.10 (pulp3), but it's still used in `create_repo` and `repo_add_updateinfo` helpers. The later one is used in two tests in `ui/test_contentview` (currently failing as the path is not used in pulp3 anymore)
```
test_positive_errata_inc_update_list_package
test_positive_composite_child_inc_update
```
CC @latran

Reply:
This would have been better as an issue, its a bit out of scope for this PR as the tests in the PR don't use those attributes. If you can get @latran to ACK it then no harm done.

Reply:
Yeah, I meant the `PULP_PUBLISHED_YUM_REPOS_PATH`. The two above being removed are not used anywhere, so would keep the PR as is.

Reply:
The two test cases are updated in another PR, but I haven't removed the filepath in that PR (thankfully) so there's no conflict from my end.  I will ack once I review your PR.